sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLoggerProviderBuilder.java

```diff
@@ -96,6 +96,24 @@ public SdkLoggerProviderBuilder addLogRecordProcessor(LogRecordProcessor process
     return this;
   }
 
+  /**
+   * Add a log processor as first. {@link LogRecordProcessor#onEmit(Context, ReadWriteLogRecord)}
+   * will be called each time a log is emitted by {@link Logger} instances obtained from the {@link
+   * SdkLoggerProvider}.
+   *
+   * <p>Compared to {@link SdkLoggerProviderBuilder#addLogRecordProcessor(LogRecordProcessor)}, this
+   * method adds the processor to the beginning of the processor pipeline. So the {@code processor}
+   * given will be executed before than the other processors already added.
+   *
+   * @param processor the log processor
+   * @return this
+   */
+  public SdkLoggerProviderBuilder addLogRecordProcessorFirst(LogRecordProcessor processor) {
```

Comment:
Can you add an equivalent method to SdkTracerProviderBuilder? Span processors have the same problem.